etcd/etcdtest/mocketcd.go

```diff
@@ -169,8 +170,16 @@ func PutData(key, data string) error {
 
 // Saves a route in etcd with the specified prefix.
 func PutDataTo(prefix, key, data string) error {
+	return PutDataToTTL(prefix, key, data, -1)
```

Comment:
Should pass zero ttl here

Reply:
Thanks!
TTL is zero now in the latest commit.
```
...
return PutDataToTTL(prefix, key, data, 0)
```